playbooks/roles/edxapp/tasks/main.yml

```diff
@@ -19,6 +19,16 @@
 # Print the shell tasks's stdout and stderr.
 - debug: var="bash_superuser.stdout_lines + [ bash_superuser.stderr ]"
 
+- name: Enable AAD
```

Comment:
I think it is safe to have it run. But we have to change through /admin panel Site to courses.microsoft.com since this SITE will be created manually and not available while this script is running and there is a Foreign Key constraint. I added necessary info to migration document already.